run3/flow/project_thnsparse.py

```diff
@@ -158,18 +159,49 @@ def check_anres(config, an_res_file, centrality, resolution,
     parser.add_argument("--outputdir", "-o", metavar="text",
                         default=".", help="output directory")
     parser.add_argument("--suffix", "-s", metavar="text",
-                        default="", help="suffix for output files")
-    parser.add_argument("--vn_method", "-vn", metavar="text",
-                        default="sp", help="vn technique (sp, ep, deltaphi)")
+                        default="", help="suffix for output files (Used for MC)") ###
```

Comment:
These informations must be added in the yaml we usually use for the postprocessing so that it is easier to point to these files and integrate everything in the `run_full` script

Reply:
Sorry, please forget this part. I wrongly added this file